stdlib/wave.pyi

```diff
@@ -37,7 +37,9 @@ class Wave_read:
     def getcomptype(self) -> str: ...
     def getcompname(self) -> str: ...
     def getparams(self) -> _wave_params: ...
+    @deprecated("Removal scheduled for Python 3.15")
```

Comment:
Would it be worth mentioning the version it's deprecated in, as well? Python 3.15 will be a long way off for people still using Python 3.8

```suggestion
    @deprecated("Deprecated in Python 3.13; removal scheduled for Python 3.15")
```

Reply:
I didn't do this in #10994, but i wonder if we should apply the decorator conditionally (under `if sys.version_info >= (3, 13)` in this case). Should users who are say on 3.9 see this warning? On the one hand, it's useful for users to know that something they're using will go away; on the other hand, if they're on an old version, the replacement may not be usable yet, so the warning is not actionable.

Reply:
Yes, I was writing up this issue as you were posting this comment :)

- #11002